include/classes/Language/Language.class.php

```diff
@@ -33,9 +33,19 @@ class Language {
 	var $status;
 	var $achecker_version;
 
+	protected $db; // Database Connection
 	// constructor
 	function Language($language_row) {
-
+		if (!isset($this->db))
```

Comment:
Instead of copying and pasting what's in DAO.class.php, can you extend Language class from DAO class?